samples/TagHelperSample.Web/Startup.cs

```diff
@@ -1,26 +1,39 @@
 ﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using Microsoft.AspNet.Builder;
 using Microsoft.AspNet.Mvc;
 using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Logging.Console;
 using TagHelperSample.Web.Services;
 
 namespace TagHelperSample.Web
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app)
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
             app.UseServices(services =>
             {
                 services.AddMvc();
-                
-                // Setup services with a test AssemblyProvider so that only the sample's assemblies are loaded. This
-                // prevents loading controllers from other assemblies when the sample is used in functional tests.
+
+                // Setup services with a test AssemblyProvider so that only the sample's assemblies are loaded. This 
+                // prevents loading controllers from other assemblies when the sample is used in functional tests. 
```

Comment:
huh?  you like trailing whitespace?  we clean this junk out every so often, try not to make it purposefully worse, and at least @harshgMSFT uses a VS extension to catch it before sending out a PR.